components/Graph.tsx

```diff
@@ -0,0 +1,150 @@
+import React, { useEffect } from 'react';
+import {
+  CartesianAxis,
+  CartesianGrid,
+  Legend,
+  Line,
+  LineChart,
+  ResponsiveContainer,
+  Tooltip,
+  XAxis,
+  YAxis,
+} from 'recharts';
+import { LegendLineGraphIcon } from '@openclimatefix/nowcasting-ui.icons.icons';
+import useSWR, { Fetcher } from 'swr';
+
+interface forecastDataPointProps {
+  target_datetime_utc: number;
+  expected_generation_kw: number;
+}
+
+interface forecastDataProps {
+  forecast_uuid: string;
+  site_uuid: string;
+  forecast_creation_datetime: number;
+  forecast_version: string;
+  forecast_values: forecastDataPointProps[];
+}
+
+interface unparsedForecastDataPointProps {
+  target_datetime_utc: string | number;
+  expected_generation_kw: number;
+}
+
+interface unparsedForecastDataProps {
+  forecast_uuid: string;
+  site_uuid: string;
+  forecast_creation_datetime: string | number;
+  forecast_version: string;
+  forecast_values: unparsedForecastDataPointProps[];
+}
+
+const fetcher: Fetcher<forecastDataProps> = async (url: string) => {
+  const tempData: unparsedForecastDataProps = await fetch(url).then((res) =>
+    res.json()
+  );
+  tempData.forecast_creation_datetime = Date.parse(
+    tempData.forecast_creation_datetime as string
+  );
+  tempData.forecast_values.map(({ target_datetime_utc }) => {
+    target_datetime_utc = Date.parse(target_datetime_utc as string);
+  });
+  return tempData as forecastDataProps;
+};
+
+const Graph = () => {
+  const { data, error, isLoading } = useSWR(
+    'http://localhost:3000/api/sites/pv_forecast/b97f68cd-50e0-49bb-a850-108d4a9f7b7e',
```

Comment:
Since we'll ultimately want this line to fetch data from their public API, maybe create an environment variable called `NEXT_PUBLIC_API_BASE_URL` and set it to `http://localhost:3000` locally. Then, in production, we can change the variable value to the real base url.

Also, the site UUID at the end of this URL will need to be dynamic since the graph should work for any site. Perhaps pass the site uuid in as a prop.

```suggestion
    `${process.env.NEXT_PUBLIC_API_BASE_URL}/api/sites/pv_forecast/${siteUUID}`,
```